percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -239,10 +239,14 @@ set_compiler(){
         export CC=/usr/bin/gcc-6
             export CXX=/usr/bin/g++-6
     fi
-    if [ x"${DEBIAN}" = xartful -o x"${DEBIAN}" = xbionic ]; then
+    if [ x"${DEBIAN}" = xbionic ]; then
         export CC=/usr/bin/gcc-7
             export CXX=/usr/bin/g++-7
     fi
+    if [ x"${DEBIAN}" = xcosmic ]; then
+        export CC=/usr/bin/gcc-8
+            export CXX=/usr/bin/g++-8
```

Comment:
maybe just fix indentation here and two lines above also

Reply:
fixed